cf-java-logging-support-servlet/src/main/java/com/sap/hcp/cf/logging/servlet/filter/CompositeFilter.java

```diff
@@ -0,0 +1,88 @@
+package com.sap.hcp.cf.logging.servlet.filter;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+
+/**
+ * <p>
+ * The {@link CompositeFilter} allows to group several servlet {@link Filter}
+ * into one. This is used to allow customizable filters and provide a backwards
+ * compatible {@link RequestLoggingFilter}.
+ * </p>
+ * 
+ * <p>
+ * You can easily create a subclass of {@link CompositeFilter} and add all the
+ * filters from com.sap.hcp.cf.logging.servlet.filter you wish. You can even
+ * bring your own filters.
+ * </p>
+ */
+public class CompositeFilter implements Filter {
+
+    private List<? extends Filter> filters;
```

Comment:
Either final **or** volatile. See comment for `setFilters(...)`.

Reply:
WIll make it final.